### PR TITLE
Add new member calls for C#, minor card refactor

### PR DESCRIPTION
**Pull request by clearab**
Add new member calls for C#, minor card refactor

### CODE REVIEW

File: samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs [hunk 1]
```diff
@@ -36,8 +38,12 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
                     await MentionActivityAsync(turnContext, cancellationToken);
                     break;
 
+                case "whoami":
```
Comment (johnataylor) [inline samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs]: inconsistent casing

File: samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs [hunk 2]
```diff
@@ -49,41 +55,68 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
...
         }
 
-        protected override async Task OnTeamsMembersAddedAsync(IList<TeamsChannelAccount> membersAdded, TeamInfo teamInfo, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
+        private async Task GetSingleMemberAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
         {
-            foreach (var teamMember in membersAdded)
-            {
-                await turnContext.SendActivityAsync(MessageFactory.Text($"Welcome to the team {teamMember.GivenName} {teamMember.Surname}."), cancellationToken);
-            }
+            TeamsChannelAccount member = await TeamsInfo.GetMemberAsync(turnContext, turnContext.Activity.From.Id, cancellationToken);
```
Comment (johnataylor) [inline samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs]: convention is var

File: samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs [hunk 3]
```diff
@@ -100,7 +133,16 @@ private async Task MessageAllMembersAsync(ITurnContext<IMessageActivity> turnCon
             var credentials = new MicrosoftAppCredentials(_appId, _appPassword);
             ConversationReference conversationReference = null;
 
-            var members = await TeamsInfo.GetMembersAsync(turnContext, cancellationToken);
+            List<TeamsChannelAccount> members = new List<TeamsChannelAccount>();
```
Comment (johnataylor) [inline samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs]: should use var

File: samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs [hunk 4]
```diff
@@ -100,7 +133,16 @@ private async Task MessageAllMembersAsync(ITurnContext<IMessageActivity> turnCon
...
+            string continuationToken = null;
+
+            do
+            {
+                TeamsPagedMembersResult currentPage = await TeamsInfo.GetPagedMembersAsync(turnContext, 100, continuationToken, cancellationToken);
```
Comment (johnataylor) [inline samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs]: use var

**Comment by clearab** [comment]
Closing, needs more thought.

**Pull request closed without merging**